src/Http/Services/FileService.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace EscolaLms\Files\Http\Services;
+
+use EscolaLms\Files\Http\Services\Contracts\FileServiceContract;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+
+class FileService implements FileServiceContract
+{
+    private string $disk;
+
+    public function __construct()
+    {
+        $this->disk = 'files';
+    }
+
+    function findAll(string $directory, array $list)
+    {
+        $ret = [];
+        /** @var UploadedFile $file */
+        foreach ($list as $file) {
+            $name = $file->getClientOriginalName();
+            $path = $directory.'/'.$name;
+            if (Storage::exists($path)) {
+                $ret []= $path;
+            }
+        }
+        return $ret;
+    }
+
+    function putAll(string $directory, array $list)
```

Comment:
Let's be consistent if there is a public function in the other files, it should be so here.